### PR TITLE
Added 'Ansible Execute Custom Role' test stub

### PR DESCRIPTION
**Pull request by rochacbruno**
Added a stub to cover the creation, availability and execution of custom ansible roles.

### CODE REVIEW

File: tests/foreman/ui/test_ansible.py [hunk 1]
```diff
@@ -191,6 +191,36 @@ def test_positive_associate_ansible_roles_to_hosts(self):
...
+           "echo 'task: shell: touch /tmp/file.test' >
+           /etc/ansible/roles/custom/"
+        2. From Web UI -> Hosts -> Select a Host -> Edit -> Ansible Roles ->
+        Select the new custom Ansible role -> Click Submit.
+        3. Run the associated Ansible role in the host.
+
+        @assert:
+
+        1. The custom role are available in Satellite.
+        1. The Ansible roles are associated to the appropriate hosts.
```
Comment (oshtaier) [inline tests/foreman/ui/test_ansible.py]: put proper numbers pls

**Comment by sthirugn** [comment]
ACK pending @oshtaier comment

File: tests/foreman/ui/test_ansible.py [hunk 2]
```diff
@@ -191,6 +191,36 @@ def test_positive_associate_ansible_roles_to_hosts(self):
...
+        2. The Ansible run reports generated.
+        3. The role task has been performed successfully in the client host
+           (ssh in to the client host and check that /tmp/file.test exists)
+
+        @caseautomation: notautomated
+        """
```
Comment (sghai) [inline tests/foreman/ui/test_ansible.py]: `@caselevel: system`

Comment (sthirugn) [inline tests/foreman/ui/test_ansible.py]: `system` is the default caselevel for this module

**Comment by sthirugn** [comment]
Looks like all changes are addressed.

**Pull request merged**